### PR TITLE
replay: ref onboarding snippet

### PR DESCRIPTION
**Pull request by bruno-garcia**
Notes sent in self review

See: https://github.com/getsentry/sentry-docs/issues/5903

### CODE REVIEW

File: src/platform-includes/session-replay/install-session-replay/javascript.mdx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Install the Replay package with NPM or your favourite package manager. Alternatively, you can load the Replay integration via a CDN bundle.
+Install the Replay package with NPM or your favorite package manager. Alternatively, you can load the Replay integration via a CDN bundle.
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply src/platform-includes/session-replay/install-session-replay/javascript.mdx:1]: TBH I'm not even sure I'd use the word "favorite" here. Maybe something more like "package manager of your choice."

Comment (bruno-garcia) [inline src/platform-includes/session-replay/install-session-replay/javascript.mdx:1]:
```suggestion
Install the Replay package with the package manager of your choice. Alternatively, you can load the Replay integration via a CDN bundle.
```

[the remark this answers is not in the document]
Comment (bruno-garcia) [reply src/platform-includes/session-replay/install-session-replay/javascript.mdx:1]: Thanks, will do

File: src/platform-includes/session-replay/setup-session-replay/javascript.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 To set up the integration, add the following to your Sentry initialization. Several options are supported and passable via the integration constructor. See the [configuration sections](/platforms/javascript/session-replay/custom-instrumentation/) for more details.
 
-```javascript {tabTitle: ESM}
+```javascript
```
[the remark this answers is not in the document]
Comment (ryan953) [reply src/platform-includes/session-replay/setup-session-replay/javascript.mdx:3]: the snippets are different

`new Replay` vs `new Sentry.Integrations.Replay`

File: src/platforms/javascript/common/session-replay/index.mdx [hunk 3]
```diff
@@ -39,9 +39,9 @@ Make sure to use the exact same version of `@sentry/replay` as your other Sentry
 
 <PlatformContent includePath="session-replay/setup-session-replay" />
 
-## Sessions
+## Replay Session
```
[the remark this answers is not in the document]
Comment (ryan953) [reply src/platforms/javascript/common/session-replay/index.mdx:42]: @jas-kas have we also used the term `user session` in this way as well?

[the remark this answers is not in the document]
Comment (jas-kas) [reply src/platforms/javascript/common/session-replay/index.mdx:42]: Yes, we use "user session" - on the Replay Index and in-app onboarding UX

(Discussed with Liza that we won't be using "Replay Session", it's the either the product name "Session Replay", or a specific "user session")

File: src/platform-includes/session-replay/setup-session-replay/javascript.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (bruno-garcia) [reply src/platform-includes/session-replay/setup-session-replay/javascript.mdx:3]: Ah! Thanks for that

File: src/platforms/javascript/common/session-replay/index.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (bruno-garcia) [reply src/platforms/javascript/common/session-replay/index.mdx:42]: Got it, will adjust

**Pull request merged**